### PR TITLE
Update a few dependencies

### PR DESCRIPTION
**Pull request by pm47**
This updates versions of a few major dependencies: scala, akka, postgres, etc.

Supersedes #2160.

### CODE REVIEW

**Review by t-bast — APPROVED** [review]
LGTM I guess, let's see if it works fine on testnet before we merge

**Comment by pm47** [comment]
I'm investigating a conflict related to akka-http versions.

**Comment by t-bast** [comment]
The usual, I feel like I already heard that during past updates :sweat_smile:

**Comment by pm47** [comment]
🧘‍♂️

File: eclair-core/src/main/scala/fr/acinq/eclair/blockchain/watchdogs/ExplorerApi.scala
Comment (pm47) [inline eclair-core/src/main/scala/fr/acinq/eclair/blockchain/watchdogs/ExplorerApi.scala:157]: That part was a bit weird.

File: eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/rpc/BasicBitcoinJsonRPCClient.scala
Comment (t-bast) [inline eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/rpc/BasicBitcoinJsonRPCClient.scala:36]: Can you explain why we choose the `WebSockets` capabilities? I don't see what we should choose, we're not actually using websockets but that `capabilites` object doesn't contain anything obvious...

File: eclair-core/src/main/scala/fr/acinq/eclair/blockchain/watchdogs/ExplorerApi.scala
Comment (t-bast) [inline eclair-core/src/main/scala/fr/acinq/eclair/blockchain/watchdogs/ExplorerApi.scala:186]: This pattern comes regularly enough that we may want to create a utility function that does this fold, don't we?
This could even be called `getBodyUnsafe` or something like that and very explicitly say that it throws?

**Review by t-bast — COMMENTED** [review]
To be honest I don't feel completely safe with these sttp changes because it's hard to know what happens under the hood, but let's test this in the wild for a while and see how it behaves.

File: eclair-core/src/main/scala/fr/acinq/eclair/blockchain/watchdogs/ExplorerApi.scala
Comment (pm47) [reply eclair-core/src/main/scala/fr/acinq/eclair/blockchain/watchdogs/ExplorerApi.scala:186]: Indeed, besides a package rename, the main change was the removal of `unsafeBody` which I replaced by this `fold`. The idea was to just "make it work" now, and later improve our code to be more in line with their functional `scodec` style.

I could use the pimp pattern to define an `unsafeBody`, but it wouldn't make the diff much smaller, due to the other changes. If you think it's worth it, I can invest some time to make our code more functional and remove those `fold` alltogether.

Comment (t-bast) [reply eclair-core/src/main/scala/fr/acinq/eclair/blockchain/watchdogs/ExplorerApi.scala:186]: No, let's keep it like that for this PR, it's good to have the dependencies updated.
That can be done in a separate PR :+1:

File: eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/rpc/BasicBitcoinJsonRPCClient.scala
Comment (pm47) [reply eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/rpc/BasicBitcoinJsonRPCClient.scala]: Done with df2adc950.

**Review by t-bast — APPROVED** [review]

**Review by t-bast — APPROVED** [review]

**Pull request merged**
